lightwood/ensemble/best_of.py

```diff
@@ -13,7 +13,8 @@
 class BestOf(BaseEnsemble):
     best_index: int
 
-    def __init__(self, target, mixers: List[BaseMixer], data: List[EncodedDs], accuracy_functions) -> None:
+    def __init__(self, target, mixers: List[BaseMixer], data: List[EncodedDs], accuracy_functions,
+                 ts_analysis: Optional[dict] = None) -> None:
```

Comment:
double check if Optional[dict] already fills in None as default

Reply:
Does not, I'm afraid